doc/md/language-manual.md

```diff
@@ -1067,7 +1067,76 @@ module {
 }
 ```
 
-On the Internet Computer, if this library is imported as identifier `Lib`, then calling `await Lib.<id>(<exp1>, …​, <expn>)`, installs a fresh instance of the actor class as an isolated IC canister, passing the values of `<exp1>`, …​, `<expn>` as installation arguments, and returns a reference to a (remote) actor of *type* `Lib.<id>`, that is, `T`. Installation is (necessarily) asynchronous.
+On the Internet Computer, if this library is imported as identifier `Lib`, then calling `await Lib.<id>(<exp1>, ..., <expn>)`, installs a fresh instance of the actor class as an isolated IC canister, passing the values of `<exp1>`, ...​, `<expn>` as installation arguments, and returns a reference to a (remote) actor of *type* `Lib.<id>`, that is, `T`. Installation is (necessarily) asynchronous.
```

Comment:
elsewhere you keep using the ellipsis unicode character

Reply:
I suggest keeping them everywhere: `s/.../…/g`

Reply:
Yeah, I opened an issue about this here: #3418. Not sure whether to use unicode or not but it should at lease be consistent throughout. I wonder if the adoc conversion went wrong...